bridge/engine/SsrcOutboundContext.h

```diff
@@ -99,32 +95,22 @@ class SsrcOutboundContext
     uint64_t lastRespondedNackTimestamp;
 
     utils::Optional<PacketCache*> packetCache;
+
+    /// ==== both Engine and Transport
+    std::atomic_uint32_t originalSsrc;
+
+    /// ==== Accessed from Engine only!
     uint64_t lastSendTime;
+    void onRtpSent(const uint64_t timestamp)
+    {
+        lastSendTime = timestamp;
+        idle = false;
+    }
 
     // Stream owner is being removed. Stop outbound packets over this context
     bool markedForDeletion;
-
     // Stream is temporarily idle and srtp encryption context is removed. It may be activated again.
     bool idle;
```

Comment:
I removed the PLI metric that appears only in logs. It was a source for data race.